src/layouts/directory.tsx

```diff
@@ -20,50 +19,33 @@ interface DirectoryGridLayoutProps {
 
 type DirectoryLayoutProps = DirectoryListLayoutProps | DirectoryGridLayoutProps;
 
-// Type guard to determine whether to render the DirectoryGrid or DirectoryList component
-// based on the type of dm_directoryChildren.
-const isDirectoryGrid = (
-  children: LocationProfile[] | DirectoryProfile<never>[]
-): children is LocationProfile[] => {
-  return children.length > 0 && "address" in children[0];
-};
-
 const DirectoryLayout = ({ data }: DirectoryLayoutProps) => {
-  const { name, dm_directoryChildren, dm_directoryParents, _site } =
-    data.document;
+  const { dm_directoryParents, dm_directoryChildren } = data.document;
 
   return (
     <>
-      <ErrorBoundaryWithAnalytics name="directory_hero">
-        <DirectoryHero title={name} brand={_site.c_brand} />
-      </ErrorBoundaryWithAnalytics>
-      <ErrorBoundaryWithAnalytics name="breadcrumbs">
-        <Breadcrumbs
-          breadcrumbs={dm_directoryParents || []}
-          separator="/"
-          className="container flex justify-center"
-          addAnalytics={true}
-        />
-      </ErrorBoundaryWithAnalytics>
-      {dm_directoryChildren && !isDirectoryGrid(dm_directoryChildren) && (
-        <ErrorBoundaryWithAnalytics name="directory">
-          <DirectoryList
-            showNumLocs={true}
-            directoryChildren={dm_directoryChildren}
-            relativePrefixToRoot={data.relativePrefixToRoot}
-          />
-        </ErrorBoundaryWithAnalytics>
-      )}
+      <DirectoryHero />
+      <Breadcrumbs
+        breadcrumbs={dm_directoryParents || []}
+        separator="/"
+        className="container flex justify-center"
+      />
       {dm_directoryChildren && isDirectoryGrid(dm_directoryChildren) && (
-        <ErrorBoundaryWithAnalytics name="directory">
-          <DirectoryGrid
-            CardComponent={DirectoryCard}
-            directoryChildren={dm_directoryChildren}
-          />
-        </ErrorBoundaryWithAnalytics>
+        <DirectoryGrid CardComponent={DirectoryCard} />
+      )}
+      {dm_directoryChildren && !isDirectoryGrid(dm_directoryChildren) && (
+        <DirectoryList showNumLocs={true} />
       )}
     </>
   );
 };
 
+// Type guard to determine whether to render the DirectoryGrid or DirectoryList component
```

Comment:
just moved this to the bottom of the file so the (more important) actual template can be at the top